### PR TITLE
Feat2

### PR DESCRIPTION
**Pull request by sherry2610**
**What changes does this PR have?**
I have integrated add feature in our app.

**Ticket :**
https://xord.atlassian.net/browse/META-217

**Is there any breaking changes?**
N/A

**Does this requires QA?**
Yes

**Steps to reproduce:**
 - Go to dashboard
  - click add button
   - enter required fields
    - submit

### CODE REVIEW

File: file4.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
 var abc = 2;
 var def = 3;
+var ghi = 4;
```
Comment (khuzama98) [inline file4.js]: please change this line

**Review by khuzama98 — CHANGES_REQUESTED** [review]
please check the requested changes

File: file4.js (hunk 1 above)
Comment (sherry2610) [reply file4.js]: Done.

**Review by khuzama98 — APPROVED** [review]
good to go.

File: file4.js (hunk 1 above)
Comment (khuzama98) [reply file4.js]: good

**Pull request merged**